server/routes/routes.go

```diff
@@ -30,6 +30,8 @@ type SystemInfo struct {
 	ManagedNamespace string `json:"managedNamespace"`
 	Namespaced       bool   `json:"namespaced"`
 	Version          string `json:"version"`
+	DisableAuth      bool   `json:"disableAuth"`
```

Comment:
This `SystemInfo` is exposed to the client, do not add those information to this struct.

Reply:
+1. I am going to create a separate struct AuthInfo to hold the `DisableAuth` and `DexServerAddr`, hence the `/systemInfo` API call won't expose auth configurations.